recipes/freealut/all/conandata.yml

```diff
@@ -0,0 +1,9 @@
+sources:
+  "1.1.0":
+    url: "http://ftp.debian.org/debian/pool/main/f/freealut/freealut_1.1.0.orig.tar.gz"
+    sha256: "60d1ea8779471bb851b89b49ce44eecb78e46265be1a6e9320a28b100c8df44f"
+patches:
+  "1.1.0":
+    - patch_file: "patches/freealut-1.1.0-conan-build.patch"
+      patch_description: "Change CMakeLists to get OpenAL from conan and allow multiplatform build as static lib"
```

Comment:
Not mandatory by any means, but I would split the CMakeLists.txt and the header modifications into two separate patches. Makes it easier to re-use the patches in case of a version bump.

Reply:
splitted